pkg/synthetictests/events.go

```diff
@@ -10,6 +10,42 @@ import (
 	"github.com/openshift/origin/pkg/test/ginkgo"
 )
 
+func combinedRegexp(arr ...*regexp.Regexp) *regexp.Regexp {
+	s := ""
+	for _, r := range arr {
+		if s != "" {
+			s += "|"
+		}
+		s += r.String()
+	}
+	return regexp.MustCompile(s)
+}
+
+var knownEvents = combinedRegexp(
+	// [sig-apps] StatefulSet Basic StatefulSet functionality [StatefulSetBasic] should not deadlock when a pod's predecessor fails [Suite:openshift/conformance/parallel] [Suite:k8s]
+	// PauseNewPods causes readiness probe to fail.
+	regexp.MustCompile(`ns/e2e-statefulset-[0-9]+ pod/ss-[0-9] node/[a-z0-9.-]+ - reason/Unhealthy Readiness probe failed: `),
+
+	// [sig-apps] StatefulSet Basic StatefulSet functionality [StatefulSetBasic] should perform rolling updates and roll backs of template modifications [Conformance] [Suite:openshift/conformance/parallel/minimal] [Suite:k8s]
+	regexp.MustCompile(`ns/e2e-statefulset-[0-9]+ pod/ss2-[0-9] node/[a-z0-9.-]+ - reason/Unhealthy Readiness probe failed: HTTP probe failed with statuscode: 404`),
```

Comment:
also intentional?

Reply:
yes, it calls `breakPodHTTPProbe`. Perhaps I need to add a commend about it